src/modules/rover_differential/module.yaml

```diff
@@ -138,6 +142,23 @@ parameters:
         decimal: 2
         default: 90
 
+      RD_MAX_THR_YAW_R:
+        description:
+          short: Yaw rate turning left/right wheels at max speed in opposite directions
+          long: |
+            This parameter is used to calculate the feedforward term of the closed loop yaw rate control.
+            The controller first calculates the required speed difference between the left and right motor to achieve the desired yaw rate.
+            This desired speed difference is then linearly mapped to normalized motor commands.
+            A good starting point is twice the speed the rover drives at maximum throttle (RD_MAX_THRTL_SPD)).
+            Increase this parameter if the rover turns faster than the setpoint, and decrease if the rover turns slower.
+        type: float
+        unit: m/s
```

Comment:
It is a bit tricky. In the code it is actually [used as m/s](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/rover_differential/RoverDifferentialControl/RoverDifferentialControl.cpp#L106-L108) because it maps the speed difference (required to achieve a certain yaw rate) between the right and left wheels to normalized motor commands.
So technically it is m/s and it is elaborated on in the _long_ description of the parameter.